File: CS/App_Code/ThemeModelBase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Xml.Serialization;

/// <summary>
/// Summary description for ThemeModelBase
/// </summary>
public class ThemeModelBase {
    string _name;
    string _title;

    [XmlAttribute]
    public string Name {
        get {
            if(_name == null)
                return "";
            return _name;
        }
        set { _name = value; }
    }
    [XmlAttribute]
    public string Title {
        get {
            if(_title == null)
                return "";
            return _title;
        }
        set { _title = value; }
    }
}

File: VB/App_Code/ThemeModelBase.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Xml.Serialization

''' <summary>
''' Summary description for ThemeModelBase
''' </summary>
Public Class ThemeModelBase
    Private _name As String
    Private _title As String

    <XmlAttribute> _
    Public Property Name() As String
        Get
            If _name Is Nothing Then
                Return ""
            End If
            Return _name
        End Get
        Set(ByVal value As String)
            _name = value
        End Set
    End Property
    <XmlAttribute> _
    Public Property Title() As String
        Get
            If _title Is Nothing Then
                Return ""
            End If
            Return _title
        End Get
        Set(ByVal value As String)
            _title = value
        End Set
    End Property
End Class